tests/cephadm/test_iscsi_multiple_removal_deployment.py

```diff
@@ -0,0 +1,81 @@
+import json
+
+from ceph.utils import get_nodes_by_ids
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class IncorrectIscsiServiceStatus(Exception):
+    pass
+
+
+def _verify_service(node, should_be_present=True):
+    """
+    Verifies iscsi service is deployed or removed
+    Args:
+        node (str): monitor node object
+        should_be_present (bool): validation option for iscsi
+    return: (bool)
+    """
+
+    # Get iscsi service status
+    out = CephAdm(node).ceph.orch.ls(service_type="iscsi", format="json-pretty")
+    log.debug(out)
+
+    # If ISCSI was deployed
+    if should_be_present:
+        service_details = json.loads(out)[0]
+        return (
+            False
+            if "service was created" not in (service_details["events"][0])
+            else True
+        )
+
+    # If ISCSI was removed
+    return False if "No services reported" not in out else True
+
+
+def run(ceph_cluster, **kw):
+    """
+    Verifies multiple deployment and removal of ISCSI services
+
+    Args:
+        ceph_cluster (ceph.ceph.Ceph): Ceph cluster object
+        kw: test data
```

Comment:
test data ? 

Add examples, it will be more clear

Reply:
added